src/Analyzers/CSharp/Tests/ConvertTypeOfToNameOf/ConvertTypeOfToNameOfFixAllTests.cs

```diff
@@ -1,269 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
I deleted this since **all** tests in the new library are fix all tests.

Tagging @sharwell to confirm.